specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2026-01-01-preview/Servers.json

```diff
@@ -0,0 +1,2185 @@
+        },
+        "dayOfWeek": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Day of the week to be used for maintenance window."
+        }
+      }
+    },
+    "Cluster": {
+      "type": "object",
+      "description": "Cluster properties of a server.",
+      "properties": {
+        "clusterSize": {
+          "type": "integer",
+          "format": "int32",
+          "default": 0,
+          "description": "Number of nodes assigned to the elastic cluster."
+        },
+        "defaultDatabaseName": {
+          "type": "string",
+          "description": "Default database name for the elastic cluster."
+        }
+      }
+    },
+    "RestartParameter": {
+      "type": "object",
+      "properties": {
+        "restartWithFailover": {
+          "type": "boolean",
+          "description": "Indicates if restart the PostgreSQL database engine should failover or switch over from primary to standby. This only works if server has high availability enabled."
+        },
+        "failoverMode": {
+          "type": "string",
+          "description": "Failover mode.",
+          "enum": [
+            "PlannedFailover",
+            "ForcedFailover",
+            "PlannedSwitchover",
+            "ForcedSwitchover"
+          ],
+          "x-ms-enum": {
+            "name": "FailoverMode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "PlannedFailover",
+                "description": "Triggers a failover from primary to standby without killing the primary database process first. This is a graceful failover that attempts to preserve data consistency."
+              },
+              {
+                "value": "ForcedFailover",
+                "description": "Kills the primary database process first, then triggers the failover. This is more aggressive and used when the primary is unresponsive or in an unhealthy state."
+              },
+              {
+                "value": "PlannedSwitchover",
+                "description": "Similar to 'PlannedFailover' but prefers a switch over operation where roles are swapped between primary and standby."
+              },
+              {
+                "value": "ForcedSwitchover",
+                "description": "Kills the primary database process and then triggers a switch over with role swapping."
+              }
+            ]
+          }
+        }
+      },
+      "description": "PostgreSQL database engine restart parameters."
+    },
+    "MigrateNetworkStatus": {
+      "type": "object",
+      "properties": {
+        "subscriptionId": {
+          "type": "string",
+          "description": "The ID of the subscription."
+        },
+        "resourceGroupName": {
+          "type": "string",
+          "description": "The name of the resource group."
+        },
+        "serverName": {
+          "type": "string",
+          "description": "The name of the server."
+        }
```

Comment:
none of these properties truly represent status of the migration - either rename MigrateNetworkStatus or add a status property if that's the intention.